--- a/src/luaconf.h	2023-05-02 22:02:30.000000000 +0200
+++ b/src/luaconf.h	2023-12-08 18:51:42.266274311 +0100
@@ -223,9 +223,9 @@
 
 #else			/* }{ */
 
-#define LUA_ROOT	"/usr/local/"
-#define LUA_LDIR	LUA_ROOT "share/lua/" LUA_VDIR "/"
-#define LUA_CDIR	LUA_ROOT "lib/lua/" LUA_VDIR "/"
+#define LUA_ROOT	"/opt/lua-" LUA_VERSION_MAJOR "." LUA_VERSION_MINOR "/"
+#define LUA_LDIR	LUA_ROOT "share/lua/"
+#define LUA_CDIR	LUA_ROOT "lib/lua/"
 
 #if !defined(LUA_PATH_DEFAULT)
 #define LUA_PATH_DEFAULT  \
