--- a/Makefile	2012-03-01 12:22:50.000000000 +0000
+++ b/Makefile	2022-06-23 10:38:30.318307730 +0000
@@ -15,7 +15,7 @@
 TARGET =            cjson.so
 PREFIX =            /usr/local
 #CFLAGS =            -g -Wall -pedantic -fno-inline
-CFLAGS =            -O3 -Wall -pedantic -DNDEBUG
+CFLAGS =            -O3 -Wall -pedantic -DNDEBUG $(LOCAL_CFLAGS)
 CJSON_CFLAGS =      -fpic
 CJSON_LDFLAGS =     -shared
 LUA_INCLUDE_DIR =   $(PREFIX)/include
